docs/dev/how-to/send-message-l2-l1.md

```diff
@@ -302,3 +203,141 @@ L1 Batch for block :>>  77512
 Retrieving proof for batch 77512, transaction index 32 and proof id 14
 Result is :>>  true
 ```
+
+## Send a message
+
+Two transactions are required:
+
+- An L2 transaction which sends a message of arbitrary length.
+- An L1 read; implemented by a getter function on an L1 smart contract.
+
+8. Get a `Contract` object that represents the [`L1Messenger`](../../reference/architecture/system-contracts.md#l1messenger) contract.
```

Comment:
Considering this part explains what is happening rather than what steps should be followed, I suggest starting the count again from 1

```suggestion
1. Get a `Contract` object that represents the [`L1Messenger`](../../reference/architecture/system-contracts.md#l1messenger) contract.
```

Reply:
Agree, fixed in the latest commit